lib/std/target.zig

```diff
@@ -109,6 +109,20 @@ pub const Target = struct {
             /// Latest Windows version that the Zig Standard Library is aware of
             pub const latest = WindowsVersion.win10_20h1;
 
+            /// Compared against build numbers reported by the runtime to distinguish win10 versions,
+            /// where 0x0A000000 + index corresponds to the WindowsVersion u32 value.
+            pub const known_win10_build_numbers = [_]u32{
+                10240, //win10
+                10586, //win10_th2
+                14393, //win_rs1
+                15063, //win_rs2
+                16299, //win_rs3
+                17134, //win_rs4
+                17763, //win_rs5
+                18362, //win_19h1
+                19041, //win_20h1
```

Comment:
You forgot 2 builds: `18363, 19042`

cf https://github.com/ziglang/zig/commit/140c599559b6e048db2ec8476cb2cc08366f6aa8#r44615067

Reply:
I thought so at first too, but those don't impact [the `NTDDI` header constant we're trying to mirror here](https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt). (Also the values are unchanged from the previous implementation.)

Reply:
This page is dated 2020-06-19, `19042` was released the 2020-10-20.
i.e. it's outdated

Reply:
Okay, it seems that you're right, [this repository's own `sdkddkver.h`](https://github.com/ziglang/zig/blob/master/lib/libc/include/any-windows-any/sdkddkver.h#L144-L146) already includes more constants than I found when I was looking for them earlier. Thank you for pointing this out!

Reply:
FYI the next one will probably be 20231 (21H1)